### ISSUE TITLE
SourceMap problem when hot reloading

### ISSUE THREAD

**Issue by lostpebble** [webpack-contrib/style-loader#235]
**What is the current behavior?**
I just updated to 0.18.0 from 0.17.0 - and now I'm getting crashing errors, like so:

```
TypeError: Cannot read property 'sourceMap' of undefined
    at addStyle (webpack:///./~/style-loader/lib/addStyles.js?:237:6)
    at addStylesToDom (webpack:///./~/style-loader/lib/addStyles.js?:107:25)
    at update (webpack:///./~/style-loader/lib/addStyles.js?:82:4)
    at eval (webpack:///./src/client/styles/entry.scss?:21:4)
    at Object.hotApply [as apply] (http://localhost:8080/dist/bundle-8-movies-breadcrumbs.js:565:16)
    at cb (webpack:///./~/webpack-hot-middleware/process-update.js?:52:36)
    at eval (webpack:///./~/webpack-hot-middleware/process-update.js?:68:13)
    at <anonymous>
From previous event:
    at check (webpack:///./~/webpack-hot-middleware/process-update.js?:67:16)
    at module.exports (webpack:///./~/webpack-hot-middleware/process-update.js?:28:5)
    at processMessage (webpack:///./~/webpack-hot-middleware/client.js?:228:9)
    at Array.handleMessage (webpack:///./~/webpack-hot-middleware/client.js?:111:7)
    at EventSource.handleMessage (webpack:///./~/webpack-hot-middleware/client.js?:74:19)
handleError @ process-update.js:116
applyCallback @ process-update.js:45
tryCatcher @ bluebird.js:5253
Promise._settlePromiseFromHandler @ bluebird.js:3275
Promise._settlePromise @ bluebird.js:3332
Promise._settlePromiseCtx @ bluebird.js:3369
(anonymous) @ bluebird.js:165
Promise resolved (async)
schedule @ bluebird.js:4481
Async.invoke @ bluebird.js:164
Promise._then @ bluebird.js:3024
Promise.then @ bluebird.js:2888
Promise.caught.Promise.catch @ bluebird.js:2869
cb @ process-update.js:59
(anonymous) @ process-update.js:68
tryCatcher @ bluebird.js:5253
Promise._settlePromiseFromHandler @ bluebird.js:3275
Promise._settlePromise @ bluebird.js:3332
Promise._settlePromise0 @ bluebird.js:3377
Promise._settlePromises @ bluebird.js:3456
(anonymous) @ bluebird.js:175
Promise resolved (async)
schedule @ bluebird.js:4481
Async.settlePromises @ bluebird.js:174
Promise._fulfill @ bluebird.js:3403
Promise._resolveCallback @ bluebird.js:3195
(anonymous) @ bluebird.js:3247
hotUpdateDownloaded @ bundle-8-movies-breadcrumbs.js:295
hotAddUpdateChunk @ bundle-8-movies-breadcrumbs.js:263
webpackHotUpdateCallback @ bundle-8-movies-breadcrumbs.js:8
(anonymous) @ :8080/dist/0.f96803f3580a00c4cdb5.hot-update.js:1
bluebird.js:1542 Unhandled rejection TypeError: Cannot read property 'sourceMap' of undefined
    at addStyle (webpack:///./~/style-loader/lib/addStyles.js?:237:6)
    at addStylesToDom (webpack:///./~/style-loader/lib/addStyles.js?:107:25)
    at update (webpack:///./~/style-loader/lib/addStyles.js?:82:4)
    at eval (webpack:///./src/client/styles/entry.scss?:21:4)
    at Object.hotApply [as apply] (http://localhost:8080/dist/bundle-8-movies-breadcrumbs.js:565:16)
    at cb (webpack:///./~/webpack-hot-middleware/process-update.js?:52:36)
    at eval (webpack:///./~/webpack-hot-middleware/process-update.js?:68:13)
    at <anonymous>
From previous event:
    at check (webpack:///./~/webpack-hot-middleware/process-update.js?:67:16)
    at module.exports (webpack:///./~/webpack-hot-middleware/process-update.js?:28:5)
    at processMessage (webpack:///./~/webpack-hot-middleware/client.js?:228:9)
    at Array.handleMessage (webpack:///./~/webpack-hot-middleware/client.js?:111:7)
    at EventSource.handleMessage (webpack:///./~/webpack-hot-middleware/client.js?:74:19)
```

**What is the expected behavior?**
Styles should reload without crashing the hot reloading

I tried updating the `css-loader` to the latest version (that was just released for seemingly the same problem) but it still crashes.

I have now reverted back to 0.17.0 and there are no problems.

**Please mention other relevant information such as your webpack version, Node.js version and Operating System.**

`"webpack": "^2.5.1",`
NodeJS v7.7.3

**Comment by lostpebble** [comment]
Sorry, see this has been mentioned on #234 already. (I had this window open for quite a while before submitting the report)

**Comment by evilebottnawi** [comment]
@lostpebble can we close this issue in favor https://github.com/webpack-contrib/style-loader/issues/234?

**Comment by michael-ciniawsky** [comment]
@lostpebble Could you try this out please (#236)  `npm i -D webpack-contrib/style-loader#fix`

**Comment by michael-ciniawsky** [comment]
Fixed by #236
